etc/scripts/json2csv.py

```diff
@@ -114,9 +114,6 @@ def collect_keys(mapping, key_group):
             if not path.endswith('/'):
                 path = path + '/'
 
-        # alway create a root directory
-        path = '/code' + path
```

Comment:
@agneet42 There should not be any sort of message. This should be a command line option that the user sets, e.g. `python json2csv.py --prepend-root in.json out.csv`